UPGRADING.rst

```diff
@@ -20,6 +20,19 @@ reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
 
+DataBiosphere/azul-private#133 Disable split tunneling for GitLab VPN in prod and anvilprod
+===========================================================================================
+
+This change requires an update to your existing VPN connections for `prod` and
+`anvilprod`. Remove these current connections from your VPN client and import
+the `.ovpn` files created by the following commands::
+
+    _select prod.gitlab  # or anvilprod.gitlab
+    cd terraform/gitlab/vpn
+    git submodule update --init easy-rsa
```

Comment:
```suggestion
```

At this point everyone must have the submodule updated.

I recommend using `git config submodule.recurse true` once in the source tree so that submodules are updated automatically on `git pull`.

Reply:
Also ask to delete the ovpn files after import so that the private key doesn't proliferate